scripts/github-actions/generate-module-index-data.js

```diff
@@ -12,15 +12,32 @@ async function getModuleDescription(
   core,
   path,
   modulePath,
+  moduleType = "brm",
```

Comment:
I'd suggest changing `moduleType` to `moduleRoot` whose value can be `modules`, `avm/res` and `avm/ptn`. Then you can eliminate `if...else if...else`.